packages/gateway/src/routes/verifyCode/handler.ts

```diff
@@ -27,8 +27,10 @@ import fetch from 'node-fetch'
 import { unauthorized } from '@hapi/boom'
 import { logger } from '@gateway/logger'
 import * as t from 'io-ts'
-import { pipe } from 'fp-ts/function'
-import { chainW, tryCatch } from 'fp-ts/Either'
+import * as F from 'fp-ts'
+
+const pipe = F.function.pipe
+const { chainW, tryCatch } = F.either
```

Comment:
just curious, is this a better approach than the previous one?

Reply:
@Zangetsu101 No, I don't think so – there was something wrong with how the TS compiler handled the import after I set 

```
"moduleResolution": "node16"
```

Might have been that the fp-ts version in use should be upgraded